kqueen/engines/openstack_kubespray.py

```diff
@@ -534,7 +552,7 @@ def _run_ansible(self, inventory="hosts.json", playbook="cluster.yml"):
         )
         pipe.wait()
         if pipe.returncode:
-            raise RuntimeError("Non zero exit status from ansible (%s)" % pipe.returncode)
+            logger.warn("Non zero exit status from ansible (%s)" % pipe.returncode)
```

Comment:
warn is deprecated, please use warning instead https://docs.python.org/3/library/logging.html#logging.Logger.warning